adoptadog-dogs.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog All Dogs</title>
+    <link rel="stylesheet" type="text/css" href="styles/adoptadog-stylesheet.css">
+    <script src="https://kit.fontawesome.com/015d63ce65.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Courgette%7CPT+Sans:700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+
+    <header>
+      <div class="grid-container">
+        <div class="grid-item-1">
+          <img src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo" style="min-width:20rem; max-width:60%;">
+          <h1 class="header-h1">Seattle's premiere dog adoption agency</h1>
+        </div>
+        <div class="grid-item-2">
+          <img src="images/cart-icon.svg" alt="Click to go to checkout" style="min-width: 2rem; max-width:5%;">$1234.56
+          <p><a href="adoptadog-checkout.html"><img src="images/checkout-button.jpg" alt="Click to go to checkout" style="min-width: 7.5rem; max-width:30%;"></a>
+        </div>
+      </div>
+    </header>
+
+    <!-- Create navigation pane -->
+    <nav class="nav-header">
+      <a class="nav-link" href="adoptadog-index.html">Home / </a>
+      <a class="nav-link" href="adoptadog-dogs.html">Dogs / </a>
+      <a class="nav-link" href="adoptadog-blog.html">Blog</a>
+    </nav>
+
+      <h1 style="padding:0rem 4rem;">Here are all of our dogs!</h1>
+
+      <!--Create flexbox grid containing dog profiles-->
+      <div class="grid-container">
+        <div class= "grid-item">
+          <div>
+          <!--Use single quotations for strings-->
+          <button onclick="dogPhotoClick('Murphy', 'American Foxhound', 123.45)"><img src="images/murphy-card.jpg" alt="Picture of Murphy" style="width:100%;"></button>
+          <script src="./js/lesson4Assignment2.js"></script>
```

Comment:
You seem to have linked the JS several times after every button. You only need to link the file once. Usually it is best to link it at end of the body, that way the html loads first and then the script, just in case it references any html elements that were not yet loaded.